src/components/screen/Friend.tsx

```diff
@@ -55,6 +67,13 @@ export default function Screen(): ReactElement {
     );
   };
 
+  if (loading) {
+    return (<Container><LoadingIndicator /></Container>)
+  }
+  if (error) {
+    return <ErrorView body={error.message} onButtonPressed={() => refetch() }/>
```

Comment:
I'd put this error handling code refer to your code.
and when I type `yarn tsc`, throws error. so here's what I fixed

```suggestion
    return <ErrorView body={error.message} onButtonPressed={(): Promise<ApolloQueryResult<Users>> => refetch()} />;
```
and also you should import `ApolloQueryResult` at top of the file
```
import { ApolloQueryResult } from 'apollo-client';
```